transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.monster.encode.transformation
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.coders.Coder
+import java.time.OffsetDateTime
+import org.broadinstitute.monster.common.{PipelineBuilder, StorageIO}
+import org.broadinstitute.monster.common.msg._
+import org.broadinstitute.monster.encode.jadeschema.table.HumanDonor
+import upack.Msg
+
+object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
+  /** (De)serializer for the upack messages we read from storage. */
+  implicit val msgCoder: Coder[Msg] = Coder.beam(new UpackMsgCoder)
+
+  /**
+    * Schedule all the steps for the Encode transformation in the given pipeline context.
+    *
+    * Scheduled steps are launched against the context's runner when the `run()` method
+    * is called on it.
+    */
+  override def buildPipeline(ctx: ScioContext, args: Args): Unit = {
+    // read in extracted info
+    val donorInputs = StorageIO
+      .readJsonLists(
+        ctx,
+        "Donors",
+        s"${args.inputPrefix}/Donor/*.json"
+      )
+
+    val humanDonorOutput = donorInputs.map(transformDonor)
+
+    // write back to storage
+    StorageIO.writeJsonLists(
+      humanDonorOutput,
+      "HumanDonors",
+      s"${args.outputPrefix}/human_donor"
+    )
+    ()
+  }
+
+  def transformDonor(donorInput: Msg): HumanDonor = {
+    HumanDonor(
+      id = donorInput.read[String]("accession"),
+      crossReferences = donorInput.read[Array[String]]("dbxrefs"),
+      timeCreated = donorInput.read[OffsetDateTime]("date_created"),
+      age = donorInput.tryRead[String]("age"),
+      ethnicities = Array() ++ donorInput.tryRead[String]("ethnicity"),
+      organism = donorInput.read[String]("organism"),
```

Comment:
Do we need `organism` when the table includes human in the name? (Its presence or absence won't directly affect cross-dataset search)

Reply:
I talked to Kathy about this and she said that she feels strongly that this will be important for cross dataset search. Maybe a thing to bring up with her if you disagree?

Reply:
This is similar to storing "status". Any hard-coded constants can be injected into the search documents at indexing time, saving us $$$ in storage. I'll talk with Kathy.

Reply:
Talked with Kathy. She thinks there's a high probability that we'll eventually be asked to ingest donor types other than "human". The least invasive way we can future-proof for that is:
1. Leave this column in here, with the hard-coded value
2. Rename the `human_donor` table to just `donor`

Reply:
Sounds good. I just pushed the changes to rename the table.